README.md

```diff
@@ -121,16 +121,7 @@ spec:
 
 ## Set up dedicated admins 
 
-To setup your cluster to have dedicated admins run the `setup/dedicated` target which installs the dedicated admin operator:
-```sh
-make setup/dedicated
-```
-
-If you want to remove the dedicated admin operator, run:
-
-```sh
-make clean/dedicated
-```
+To setup your cluster to have dedicated admins run the `./scripts/setup-htpass-idp.sh` script which installs the dedicated admin operator and creates users.
```

Comment:
I think dedicated admin operator is installed by default, so this script only creates users